src/views/SaveCredential/SaveCredential.tsx

```diff
@@ -67,40 +66,31 @@ export function SaveCredential(): JSX.Element | null {
     <main className={styles.container}>
       <h1 className={styles.heading}>{t('view_SaveCredential_heading')}</h1>
 
-      <dl className={styles.details}>
-        <dt className={styles.detailName}>Credential type:</dt>
-        <dd className={styles.detailValue}>{credential.cTypeTitle}</dd>
-
-        <dt className={styles.detailName}>Attester:</dt>
-        <dd className={styles.detailValue}>{credential.attester}</dd>
-      </dl>
-
-      <h2 className={styles.warning}>{t('view_SaveCredential_warning')}</h2>
-
-      <label className={styles.label}>
-        {t('view_SaveCredential_name')}
-        <input
-          name="name"
-          className={styles.name}
-          onInput={handleNameInput}
-          autoFocus
-          defaultValue={credential.name}
-        />
-      </label>
-
-      <p className={styles.buttonsLine}>
-        <button type="button" className={styles.cancel} onClick={handleCancel}>
-          {t('common_action_cancel')}
-        </button>
-        <a
-          download={download.name}
-          href={download.url}
-          className={styles.submit}
-          aria-disabled={!credential.name || credential.name.length === 0}
-        >
-          {t('view_SaveCredential_CTA')}
-        </a>
-      </p>
+      <section className={styles.cardContainer}>
+        <CredentialCard credential={credential} expand buttons={false} />
+      </section>
+
+      <h2 className={cx(styles.warning, { [styles.downloaded]: isDownloaded })}>
```

Comment:
I think doing it like this (and adjusting the CSS accordingly) does not complicate the code, while keeping us protected from an incorrect order of CSS rules.
```suggestion
      <h2 className={isDownloaded ? styles.downloaded : styles.warning}>
```